ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml

```diff
@@ -40,6 +40,66 @@ tests:
     test:
     - chain: openshift-qe-cluster-density-v2
     workflow: openshift-qe-installer-azure-aro
+- as: ocp-qe-perfscale-ci-tests-aro-node-density-small-scale
```

Comment:
Please update the job names to `ocp-qe-perfscale-ci-tests-node-density-24nodes-aro`

Reply:
You can combine `node-density` & `cluster-density-v2` two jobs into 1  with this [chain](https://steps.ci.openshift.org/chain/openshift-qe-ocp-qe-perfscale-ci-tests)

Reply:
Done